test/backend/datastores.mock.spec.js

```diff
@@ -15,19 +15,21 @@ import {
 
 
 let servers;
-let connectionId;
+let connectionId = 'postgres-189ebfb4-e1b4-446c-9b83-b326875fa2d8';
+let storagePath;
 
 describe('Datastore Mock:', function () {
     beforeEach(() => {
         servers = createTestServers();
+        storagePath = process.env.PLOTLY_CONNECTOR_STORAGE_PATH;
+        process.env.PLOTLY_CONNECTOR_STORAGE_PATH = 'mock-storage'
         saveSetting('AUTH_ENABLED', false);
-        sqlConnections.mock = true;
-        connectionId = saveConnection(sqlConnections);
     });
 
     afterEach(() => {
-        sqlConnections.mock = false;
-        connectionId = saveConnection(sqlConnections);
+        clearSettings('SETTINGS_PATH');
```

Comment:
Is `clearSettings('SETTINGS_PATH');` needed? (`createTestServers();` does `clearSettings('CONNECTIONS_PATH', 'QUERIES_PATH', 'SETTINGS_PATH');`)

Reply:
> Is clearSettings('SETTINGS_PATH'); needed? 

Yes, otherwise "mock-storage/settings.yaml" will be created on every test run.

Reply:
I don't follow the reasoning: `clearSettings('SETTINGS_PATH');` removes `mock-storage/settings.yaml`.

I've just tested `test/backend/datastores.mock.spec.js` without `clearSettings('SETTINGS_PATH');` and `saveSetting('AUTH_ENABLED', true);` and the specs pass.

Reply:
Ok, Removed the lines..!!